framework/src/libtorch/userobjects/TorchScriptUserObject.C

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifdef LIBTORCH_ENABLED
+
+#include "TorchScriptUserObject.h"
+
+registerMooseObject("MooseApp", TorchScriptUserObject);
+
+InputParameters
+TorchScriptUserObject::validParams()
+{
+  InputParameters params = GeneralUserObject::validParams();
+  params.addClassDescription("User-facing object which loads a torch script module.");
+  params.addRequiredParam<std::string>("filename",
```

Comment:
Shouldn't this be `FileName` instead of `std::string`?

Reply:
Oh yes, great catch!